src/adcm_pytest_plugin/docker_utils.py

```diff
@@ -296,17 +305,19 @@ def adcm_container(
         Return ADCM container and bind port.
         """
         for _ in range(0, CONTAINER_START_RETRY_COUNT):
-            port = _find_random_port(ip)
+            api_port = "8000/tcp"
```

Comment:
In this case i don`t we any need in for try except construction.

Reply:
@Maunty, thank you, done

Reply:
@madlabman `class RetryCountExceeded` may be removed too but it is backward compatibility broke. Consult with @dgusakov .

Reply:
Removed